### ISSUE TITLE
Support WYSIWYG editor

### ISSUE THREAD

**Issue by mrahmadt** [InfyOmLabs/laravel-generator#870]
Hello Team,

I tried to search for this in the online documentation without any result, is it possible to auto generate WYSIWYG field with laravel generator or I have to code it myself?

**Comment by mitulgolakiya** [comment]
@machour no, right now it doesn't support that kind of field. You will need to add that manually after you generate the CRUD.
